addon/styles/_frost-expand.scss

```diff
@@ -0,0 +1,56 @@
+$frost-color-expand-label-text: $frost-color-blue-1 !global;
+$frost-color-expand-scroll-border: $frost-color-lgrey-3 !global;
+$frost-color-expand-content-bg: $frost-color-lgrey-4 !global;
+$frost-color-expand-content-text: $frost-color-grey-3 !global;
+
+.frost-expand {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: $frost-font-family;
+
+  &.collapsed {
+    .frost-expand-label-icon {
+      transform: rotate(-90deg);
+    }
+  }
+
+  &.expanded {
+    .frost-expand-label-icon {
+      transform: rotate(0deg);
+    }
+  }
+
+  &-label {
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    justify-content: flex-start;
```

Comment:
I'm fine with flex, but I think it would make sense to add `align-items: center` to make the behaviour explicit (and guard against a different height text/icon)